src/routers/AppRouter.tsx

```diff
@@ -0,0 +1,12 @@
+import { Routes, Route, BrowserRouter } from 'react-router-dom'
+import { Dashboard } from '../components/Dashboard'
+import { LoginPage } from '../components/LoginPage'
+
+export const AppRouter = () => (
+  <BrowserRouter>
+    <Routes>
+      <Route path="/" element={<LoginPage />} />
+      <Route path="/dashboard" element={<Dashboard />} />
```

Comment:
We would need `Private` and `Public` routes in the future. Could you add a task for that?

Reply:
Task created